layouts/404.html

```diff
@@ -16,16 +16,16 @@
           <div class="jumbotron text-center mb-0">
             <img class="img-fluid" src="{{ printf "images/logo.svg" | relURL }}"></img>
             <h1 class="jumbotron-heading my-5">
-              {{- i18n "404.title" -}}
+              {{ i18n "404.title" }}
             </h1>
             <h3 class="my-4">
-              {{- i18n "404.subtitle" -}}
+              {{ i18n "404.subtitle" }}
             </h3>
             <h5 class="my-4">
-              {{- i18n "404.direction" -}}
+              {{ i18n "404.direction" }}
             </h5>
             <a class="btn btn-primary btn-lg my-4" href="/">
-              {{- i18n "404.button" -}}
+              {{ i18n "404.button" }}
```

Comment:
Why the change to not remove whitespace here?

Reply:
Oh, I removed this here because I created a branch off my previous PR and had the changes reflect here I thought it will cause some kinda merge conflict. I'll add it back.